### DISCUSSION TITLE
Flax requires jax>=0.2.21 but unable to install it

### DISCUSSION THREAD

**Discussion by jrabary** [jax-ml/jax#8399] [General] [answered]
Dear all,

I've already asked this question on the Flax repository : https://github.com/google/flax/discussions/1651.

When installing jax with `pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html` I always get the version `0.2.17` of jax but flax need the version `0.2.21`.

Is there a quick workaround for this problem ?

Best regards,

Jao

**Comment by hawkinsp** [comment, marked as answer]
I suspect the reason is that you are using Python 3.6. We dropped support for Python 3.6 in jax v0.2.18, per the [deprecation policy](https://jax.readthedocs.io/en/latest/deprecation.html).

Please update to a supported version of Python.

**Comment by jrabary** [comment]
Indeed my python version is 3.6.9. Thanks!
